frontend/src/elm/Pages/Lobby.elm

```diff
@@ -1,19 +1,21 @@
 module Pages.Lobby exposing (..)
```

Comment:
general comment for this file: let us not use names like cmd, cmd1, cmd2; but try to use more meaningful name like saveToWebstorageCmd